server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -4237,6 +4245,260 @@ public VlanVO doInTransaction(final TransactionStatus status) {
 
     }
 
+    @Override
+    public Vlan updateVlanAndPublicIpRange(UpdateVlanIpRangeCmd cmd) throws ConcurrentOperationException,
```

Comment:
I think I mentioned that before, but shouldn't this be in the `IpAddressManager`?

Reply:
@DaanHoogland 
createVlanAndPublicIpRange and deleteVlanAndPublicIpRange are in this file.

Reply:
all of them should go to IpAddressManager :( (not in this PR probably)